docs/JSON-RPC-API-using-jmwalletd.md

```diff
@@ -0,0 +1,104 @@
+## JSON-RPC API for Joinmarket using jmwalletd.py
+
+### Introduction - how to start the server
+
+Create an ssl certificate and store it in `<datadir>/ssl/{key,cert}.pem`; the `datadir` is set by `--datadir` in scripts or is `~/.joinmarket` by default, or `.` by default in testing.
+
+After installing Joinmarket as per the [INSTALL GUIDE](INSTALL.md), navigate to the `scripts/` directory as usual and start the server with:
+
+```
+(jmvenv) $python jmwalletd.py
+```
+
+which with defaults will start serving the RPC over `https://` on port 28183, and a (secure) websocket server (`wss://`) on port 28283.
+
+Documentation of the websocket functionality [below](#websocket).
+
+This HTTP server does *NOT* currently support multiple sessions; it is intended as a manager/daemon for all the Joinmarket services for a single user. Note that in particular it allows only control of *one wallet at a time*.
+
+#### Rules about making requests
+
+Authentication is with the [JSON Web Token](https://jwt.io/) scheme, provided using the Python package [PyJWT](https://pypi.org/project/PyJWT/).
+
+Note that for some methods, it's particularly important to deal with the HTTP response asynchronously, since it can take some time for wallet synchronization, service startup etc. to occur; in these cases a HTTP return code of 202 is sent.
+
+### API documentation
+
+Current API version: v1.
+
+The [OpenAPI](https://github.com/OAI/OpenAPI-Specification) spec is given in [this yaml file](../jmclient/jmclient/wallet-rpc-api.yaml). Human readable documentation of the API is provided in [this document](../jmclient/jmclient/wallet-rpc-api.md), which is auto-generated with the node utility [swagger-markdown](https://www.npmjs.com/package/swagger-markdown).
+
+Those wishing to write client code should adhere to that specification.
+
+#### What is and is not provided in the current version of the API.
+
+As a brief summary, the functionality currently available is:
+
+* create a wallet
+* unlock (decrypt) a wallet
+* lock a wallet
+* display contents of a wallet
+* list the utxos in the wallet
+* get a new address for deposit in a given account
+* send a payment without coinjoin
+* send a paymen with coinjoin
```

Comment:
```suggestion
* send a payment with coinjoin
```

Reply:
Done in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/996/commits/938aa7659ff44d7621294e44bb8a244ccf1b00dd